### DISCUSSION TITLE
None of the event listener working

### DISCUSSION THREAD

**Discussion by Mubashir1234** [remix-run/remix#9607] [Q&A]
hi there I am trying to listen an events on various platforms. but none of the event listeners are listing on the remix I had already imported the Script tagin body but I still the same issue.

here is one of the example:

 <div onClick={()=> console.log('hello')} className="col-span-4 md:col-span-4 lg:col-span-3 flex justify-end items-center">
                  <input type="checkbox" value="" className="sr-only peer" checked={false} />
                  <div className="relative w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-blue-300 dark:peer-focus:ring-blue-800 rounded-full peer dark:bg-gray-700 peer-checked:after:translate-x-full rtl:peer-checked:after:-translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:start-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-gray-600 peer-checked:bg-black"></div>
                  </div>

  this is my root.TSX file 
import {
  Links,
  LiveReload,
  Meta,
  Outlet,
  Scripts,
  ScrollRestoration,
} from "@remix-run/react";
import styles from "./input.css"
import { LinksFunction } from "@remix-run/node";

export function Layout({ children }: { children: React.ReactNode }) {
  return (
    <html lang="en">
      <head>
        <meta charSet="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <Meta />
        <Links />
        <script src="http:localhost:5173" async></script>
      </head>
      <body>
     
        {children}
        <ScrollRestoration />
        <Scripts />
      </body>
    </html>
  );
}

export default function App() {
  return(
    <>
     <Outlet />;
    </>
  )

**Comment by kiliman** [comment]
Not sure what happened to your code sample, but it's illegible.

Can you create a simple repro on ⚡️ StackBlitz https://remix.new
